src/carousel.scss

```diff
@@ -0,0 +1,155 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-carousel;
+$button: #{$fd-namespace}-button;
+
+.#{$block} {
+  $fd-carousel-page-indicator-border: 0.0625rem solid var(--sapPageFooter_BorderColor) !default;
+  $fd-carousel-button-size: 2.125rem !default;
+  $fd-carousel-button-content-offset: 0.5rem !default;
+  $fd-carousel-dot-size: 0.25rem !default;
+  $fd-carousel-dot-active-size: 0.5rem !default;
+
+  @include fd-reset();
+  @include fd-flex(column);
+
+  width: 100%;
+  min-width: 15.5rem;
+  max-width: 100%;
+  height: 100%;
+
+  &__content {
+    @include fd-reset();
+
+    flex: 1 1 auto;
+    width: 100%;
+    background: var(--sapBackgroundColor);
+    position: relative;
+
+    .#{$block}__button.#{$button} {
+      box-shadow: var(--sapContent_Shadow1);
+      position: absolute;
+      top: calc(50% - 1.0625rem);
```

Comment:
What is `1.0625rem`? Is this half of the button size?
We can also position it like that to get rid of this value:
```
      top: calc(50%);
      transform: translateX(-50%);
```